healvis/tests/test_visibility.py

```diff
@@ -77,15 +80,15 @@ def test_vis_calc():
     shell[ind] = 1  # Jy/pix
     shell[ind] *= visibility.jy2Tstr(freqs[0], bm=pix_area)  # K
 
-    obs = visibility.observatory(latitude, longitude, array=[bl], freqs=freqs)
+    obs = visibility.Observatory(latitude, longitude, array=[bl], freqs=freqs)
     obs.pointing_centers = centers
     obs.times_jd = np.array([1])
     obs.set_fov(fov)
     obs.set_beam('uniform')
 
     visibs, times, bls = obs.make_visibilities(shell)
     print(visibs)
-    nt.assert_true(np.real(visibs) == 1.0)  # Unit point source at zenith
+    nt.assert_true(np.isclose(np.real(visibs), 1.0).all())  # Unit point source at zenith
```

Comment:
Ah, no it was actually raising an Error because visibs is an an ndarray, and `nt.assert_true` only takes booleans as argument. By exact, do you mean within float precision, b/c that is basically what the default of `np.isclose` does, or do you mean literally the exact same bits? I think probably we want a call to `isclose`